WordPress/twentytwentyone
# Add visible focus indicator to linked images

Fixes https://github.com/WordPress/twentytwentyone/issues/728

## Summary


Adds a generic visible focus indicator to linked images.
The offset is different depending on the block.

## Relevant technical choices:

## Test instructions

This PR can be tested by following these steps:
1. Add one of each block listed above. Legacy images can be tested with the theme unit test.
1. Make sure there are links added to the the images.
1. Focus on the links 
1. Make sure that there is visible focus style.


## Screenshots

Media & text, with and without border:
![linked-media-focus](https://user-images.githubusercontent.com/7422055/97711380-5446cf80-1abd-11eb-859e-2280314765be.png)

Gallery block, gallery item:
![linked-gallery-item-focus](https://user-images.githubusercontent.com/7422055/97711408-5f99fb00-1abd-11eb-88a5-e5420e73ed06.png)

Classic/legacy gallery item: 
![classic-gallery-item-focus](https://user-images.githubusercontent.com/7422055/97711459-67599f80-1abd-11eb-8d09-a15bc5fd1ecb.png)

Image block:
![linked-image-block-focus](https://user-images.githubusercontent.com/7422055/97711515-79d3d900-1abd-11eb-9028-6e1223dce78e.png)


## Quality assurance
